mtgjson5/price_builder.py

```diff
@@ -159,10 +159,14 @@ def build_today_prices() -> Dict[str, Any]:
     tcgplayer = _generate_prices(TCGPlayerProvider())
     card_market = _generate_prices(CardMarketProvider())
     card_kingdom = _generate_prices(CardKingdomProvider())
+    tcgplayer_buylist = _generate_tcgplayer_buylist_prices(TCGPlayerProvider())
```

Comment:
I don't agree with making a new function. We have an internal API (read: interface) for a reason. If you could make it fit that model, would be more likable